gitops/snapshot.go

```diff
@@ -77,6 +77,9 @@ const (
 	// SnapshotGitSourceRepoURLAnnotation contains URL of the git source repository (usually needed for forks)
 	SnapshotGitSourceRepoURLAnnotation = "test.appstudio.openshift.io/source-repo-url"
 
+	// SnapshotGitSourceRepoBranchAnnotation contains branch of the git source repository (usually needed for forks)
+	SnapshotGitSourceRepoBranchAnnotation = "test.appstudio.openshift.io/source-branch"
```

Comment:
I see this new constant being added, where is it being used exactly?

Reply:
no, 
was not used in any place,  I forgot to clean it after debugging , removed
thx